package.json

```diff
@@ -144,18 +144,20 @@
   },
   "devDependencies": {
     "@types/node": "^13.5.0",
-    "@types/semver": "^6.2.0",
     "coc.nvim": "0.0.74",
+    "npm-run-all": "^4.1.5",
     "parcel-bundler": "^1.12.4",
     "prettier": "1.19.1",
     "rimraf": "^3.0.0",
     "ts-loader": "^6.0.3",
     "typescript": "3.7.5"
   },
   "dependencies": {
-    "metals-languageclient": "^0.1.10",
+    "metals-languageclient": "0.1.11",
     "promisify-child-process": "^3.1.3",
-    "semver": "^7.1.1",
     "vscode-languageserver-protocol": "^3.15.1"
+  },
+  "resolutions": {
+    "vscode-jsonrpc": "^5"
```

Comment:
`coc.nvim` and `vscode-languageserver-protocol` use different versions of this dependency which was causing the typing issue on `LanguageClient` (temporarily fixed by `any`).

This pins it to the more recent one, which makes the error go away.

It still needs some manual testing to make sure everything works as expected.

Reply:
Sounds good. Thanks for hunting this down. Once this is rebased, I'll pull this down and do some testing.